src/views/tutorial-library/components/pagination/index.tsx

```diff
@@ -0,0 +1,137 @@
+import { IconArrowLeft16 } from '@hashicorp/flight-icons/svg-react/arrow-left-16'
+import { IconArrowRight16 } from '@hashicorp/flight-icons/svg-react/arrow-right-16'
+import ButtonLink from 'components/button-link'
+import Button from 'components/button'
+import { usePagination } from 'react-instantsearch-hooks-web'
+
+import paginationStyle from './pagination.module.css'
+import { ButtonLinkProps } from 'components/button-link/types'
+
+/**
+ * @TODO de-duplicate some of the link generation, especially on either side of the pages array
+ * @TODO scroll to top of page after paging
+ */
+export function Pagination() {
```

Comment:
This component is very skateboard-y 🛹

Acknowledging that there is an HDS Pagination component on the way, I didn't spend a ton of time making this as polished as I could have.